patches/minecraft/net/minecraft/entity/item/BoatEntity.java.patch

```diff
@@ -1,6 +1,47 @@
 --- a/net/minecraft/entity/item/BoatEntity.java
 +++ b/net/minecraft/entity/item/BoatEntity.java
-@@ -459,7 +459,7 @@
+@@ -49,6 +49,7 @@
+    private static final DataParameter<Integer> field_184462_b = EntityDataManager.func_187226_a(BoatEntity.class, DataSerializers.field_187192_b);
+    private static final DataParameter<Float> field_184464_c = EntityDataManager.func_187226_a(BoatEntity.class, DataSerializers.field_187193_c);
+    private static final DataParameter<Integer> field_184466_d = EntityDataManager.func_187226_a(BoatEntity.class, DataSerializers.field_187192_b);
++   private static final DataParameter<String> BOAT_TYPE_NAME = EntityDataManager.func_187226_a(BoatEntity.class, DataSerializers.field_187194_d);
```

Comment:
Are you sure vanilla doesn't have Problems with another Data Parameter...?

Reply:
I have checked a map generated without these changes. But with Forge. Maybe Vanilla acts different.